lib/kdump_utils.pm

```diff
@@ -132,6 +138,10 @@ sub activate_kdump {
         wait_screen_change { send_key 'alt-o' } if match_has_tag('yast2-kdump-enabled');
         wait_screen_change { send_key 'alt-o' } if match_has_tag('yast2-kdump-restart-info');
         wait_screen_change { send_key 'alt-i' } if match_has_tag('yast2-missing_package');
+        # problem with e2fsprogs requirement on JeOS
+        if (match_has_tag('yast2-kdump-unexpected-issue')) {
+            die 'Missing e2fsprogs requirement, bsc#1140040';
+        }
```

Comment:
Redundant comment, since the error message already state the reason.
The 'if' statement can be done in line, to follow the previous statements style and easy readability.

```perl
die('Missing e2fsprogs requirement, bsc#1140040') if (match_has_tag('yast2-kdump-unexpected-issue'));
```